src/configurations/destinations/clevertap_segment/ui-config.json

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
Could you please add appropriate note to all the fields instead of leaving it blank